app/models/gobierto_common/custom_field_value/localized_text.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module GobiertoCommon::CustomFieldValue
+  class LocalizedText < Base
+    def value
```

Comment:
I don't understand what does `super` in this method

Reply:
Yes, its useless